src/applications/lgy/coe/status/README.md

```diff
@@ -0,0 +1,140 @@
+App Name: `Certificate of Eligibility (COE) Status App`
+Active engineers: 
+Form ID (if different from app name, NA otherwise): `NA`
+URL: `{root url}/housing-assistance/home-loans/check-coe-status/your-coe`
+
+
+# Background
+When a Veteran wants to get a VA backed home loan the VA can provide them with something called a "Certificate of Eligibility" (COE). This certificate does not approve them for a specific dollar amount but rather lets the Veteran's chosen lender know that they are in fact able to get a VA backed home loan.
+
+When a veteran wants to recieve a COE they can come to this page and we can make a call to the upstream data service LGY and they can send us a determination of if the Veteran is automatically approved for a COE. If they are automatically approved then we can provide them a link to download that COE. Alternatively if the Veteran applied for a COE then the call we make the LGY serivice will return to us the status of that COE application that the Veteran has in progress.
+
+## How the app works for the user
+When a Veteran first lands on the page we make the initial call to the LGY service to get the status of the COE for the Veteran. We then show content relevent to that Veteran based on the status that comes back from LGY, so for example if the Veteran is automatically approved for a COE we show them a message that says that they are automatically approved as well as a link to download the COE as a PDF.
+
+## The front end code
+The front end for the COE status app is broken up into `containers` and `components` with the main entry point of the app being `app-entry.jsx`. Inside the `containers` directory there is a file called `App.js` which housese the main logic for the front end, everything else are components rendered inside this main `App.js` file. We use Redux to make the call to the LGY endpoints for the status of the COE, those are one level out in a `/shared` folder since they are used in this app and in the form app.
+
+Inside `App.js` we import the Redux actions and based on the data returned by the LGY service called in those Redux actions we enter into a block of logic. That logic first tests if the call to LGY is pending and if it is it returns a loading indicator. Once the call to LGY is successful we enter into a switch statement based on the data returned by the LGY service. 
+
+
+### Statuses
```

Comment:
Do we want to include this here? This level of detail seems a bit much for frontend considerations

Reply:
I removed the portion of back end code but I think the rest is still good, what do you think?